labman/db/study.py

```diff
@@ -114,18 +114,21 @@ def specimen_id_to_sample_id(self, specimen):
 
         with sql_connection.TRN as TRN:
             if specimen_id_column is None:
+                # assuming specimen_id_column should be sample_id column,
+                # verify that the sample_id exists in study_sample.
                 sql = """SELECT sample_id
                          FROM qiita.study_sample
                          WHERE
                          sample_id = %s
                          """
             else:
                 sql = """SELECT sample_id
-                         FROM qiita.sample_{0}
+                         FROM qiita.sample_{0} as {1}
```

Comment:
I am not very familiar with jsonb conventions, why is the aliasing of the table name necessary here?

Reply:
Testing these queries with the sample database load, I noticed that the resulting table names this column something like '\?column?/' by default. I aliased this to call attention to it, and have the results table be more in line with what we'd normally expect, just in case we have to use it as input for another query, etc.